cumulus/zombienet/zombienet-sdk/README.md

```diff
@@ -2,18 +2,21 @@
 
 As a prerequisite, the `test-parachain` and `polkadot` binaries need to be installed or available under `$PATH`.
 
+The following commands need to be run from the repository root:
 ```
 # install test-parachain
-cargo install --path ./cumulus/test/service --locked --release
+cargo install --path ./cumulus/test/service --locked
```

Comment:
Why did you remove `--release` here but it is still in run.sh script?

Reply:
`--release` is not an available CLI command for `install`. Also `release` is the default profile AFAIK.